auditable-serde/src/lib.rs

```diff
@@ -455,4 +461,50 @@ mod tests {
         let version_info_struct: VersionInfo = (&metadata).try_into().unwrap();
         let _lockfile_struct: cargo_lock::Lockfile = (&version_info_struct).try_into().unwrap();
     }
+
+    #[cfg(feature = "schema")]
+    /// Generate a JsonSchema for VersionInfo
+    fn generate_schema() -> schemars::schema::RootSchema {
+        let mut schema = schemars::schema_for!(VersionInfo);
+        let mut metadata = *schema.schema.metadata.clone().unwrap();
+
+        let title = "cargo-auditable schema".to_string();
+        let id = "https://raw.githubusercontent.com/rust-secure-code/cargo-auditable/master/cargo-auditable.schema.json"
```

Comment:
Should this point to the current tag/branch/commit?

Reply:
I've now read the spec at https://json-schema.org/draft/2020-12/json-schema-core.html#name-the-id-keyword. This doesn't need to be a network accessible URL - I've updated to not refer to github/git repos.